### PR TITLE
AD, PT: Fix radio button styling

### LINKED ISSUE #107: Radio buttons are inconsistent between browsers
### Chrome 

 chrome-radio

### Firefox

firefox-radio

### Safari

safari-radio

## Expected behavior/Screenshot:

 - They should all be consistent and correct across all supported browsers
 - Both the hover and active states should be perfectly central within the button without any clipping

@peterthompson @alexnaish

### PR DESCRIPTION
**Pull request by nowarkdev**
## Fixes
#107 

## What this PR resolves
Radio button styling now consistent between browsers.

## Screenshot after PR:
screen shot 2016-02-24 at 16 10 50

### CODE REVIEW

File: src/components/radio.scss [hunk 1]
```diff
@@ -6,48 +6,68 @@ $icon-path: './icons' !default;
...
 
+
+//Box
+//Border
+//Background
+//Text
+//Other
+
 .now-radio + label {
+  box-sizing: border-box;
```
Comment (peter-mouland) [inline src/components/radio.scss]: I'm sure `box-sizing: border-box` is by default, so don't think this is needed anymore

**Comment by peter-mouland** [comment]
great screenshot :)

File: src/components/radio.scss (hunk 1 above)
Comment (nowarkdev) [inline src/components/radio.scss]: Just looked it up on MDN and `content-box` is the default value. So, I think we still need `border-box`.

@PLAGR34GTT @andhapp

**Comment by nowarkdev** [comment]
Rebased against master and squashed to one commit.

@PLAGR34GTT @andhapp

File: src/components/radio.scss [hunk 2]
```diff
@@ -7,47 +7,60 @@ $icon-path: './icons' !default;
 }
 
 .now-radio + label {
+  box-sizing: border-box;
```
Comment (peter-mouland) [inline src/components/radio.scss]: I've just taken a look and our style-guide updates everything to border box:
https://github.com/sky-uk/nowtv-styleguide/blob/master/src/base/_base.scss#L3-L5

**Comment by rogerfranklin2** [comment]
:+1:

**Pull request merged**